### ISSUE TITLE
Bootstrap-vue unit test not working with tooltip directive

### ISSUE THREAD

**Issue by stripathix** [bootstrap-vue/bootstrap-vue#5567]
### Describe the bug

Unit test not working with tooltip directive

### Steps to reproduce the bug
Using a very basic example. Running this unit test case fails every time. `aria-describedby` is always undefined. Note I have used spec file https://github.com/bootstrap-vue/bootstrap-vue/blob/dev/src/directives/tooltip/tooltip.spec.js for reference.
```

import { mount } from '@vue/test-utils';
import { VBTooltip } from 'bootstrap-vue/src/directives/tooltip/tooltip';
import demand from 'must';

const createContainer = (tag = 'div') => {
   const container = document.createElement(tag)
   document.body.appendChild(container)
   return container
}

const waitNT = ctx => new Promise(resolve => ctx.$nextTick(resolve))
const waitRAF = () => new Promise(resolve => requestAnimationFrame(resolve))

describe('TkDownloadButton', () => {
  const App = {
    template: '<button v-b-tooltip.click.html title="<b>foobar</b>">button</button>'
  }
  it('must show interpolated message', async () => {
    const wrapper = mount(App, {
      directives: {
        bTooltip: VBTooltip
      },
      attachTo: createContainer()
    })
    await wrapper.vm.$nextTick();
    await waitNT(wrapper.vm)
    await waitRAF()
    await waitNT(wrapper.vm)
    await waitRAF()
    await waitNT(wrapper.vm)
    await waitRAF()
    await waitNT(wrapper.vm)
    await waitRAF()
    const BV_TOOLTIP = '__BV_Tooltip__';
    const $button = wrapper.find('button')
    demand($button.attributes('aria-describedby')).to.be.undefined();
    await $button.trigger('click')
    await waitRAF()
    await waitRAF()
    await waitNT(wrapper.vm)
    await waitRAF()
   // ARIA-DESCIBED IS UNDEFINED HERE BUT IT IS SHOULD HAVE SOME VALUE
    console.log($button.attributes('aria-describedby'))
    const adb = $button.attributes('aria-describedby');
    const tip = document.querySelector(`#${adb}`);
    demand(tip).not.to.be(null);
    demand(tip.classList.contains('tooltip')).to.be(true);
  });
});
```

### Versions

**Libraries:**

- BootstrapVue: 2.15.0
- Bootstrap: 4.5.0
- Vue: 2.6.11
- @vue/test-utils: 1.0.3

**Environment:**

- Device: Windows 10 laptop
- OS: Window
- Browser: Chrome

### Additional context
Needed to add a unit test to check if the tooltip is bound right ith a button. It was working fine with below versions
```
    "@vue/test-utils": "1.0.0-beta.28",
    "bootstrap": "4.3.1",
    "bootstrap-vue": "2.0.0-rc.16",
```
But since I updated to the latest aria-described is always undefined. Looks like tooltip directive is not having any effect on the element.
